feature/qos/tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -900,35 +977,31 @@ func testQoSCiscoClassifierConfig(t *testing.T) {
 			return out
 		})
 
-		if tc.classType == oc.Qos_Classifier_Type_IPV4 {
+		if tc.name == "dscp_based_classifier_ipv4" {
 			if equal := cmp.Equal(gnmi.GetConfig(t, dut, condition.Ipv4().DscpSet().Config()), tc.dscpSet, trans); !equal {
 				t.Errorf("condition.Ipv4().DscpSet().State(): got %v, want %v", gnmi.GetConfig(t, dut, condition.Ipv4().DscpSet().Config()), tc.dscpSet)
 			}
-		} else if tc.classType == oc.Qos_Classifier_Type_IPV6 {
+		} else if tc.name == "dscp_based_classifier_ipv6" {
 			if equal := cmp.Equal(gnmi.GetConfig(t, dut, condition.Ipv6().DscpSet().Config()), tc.dscpSet, trans); !equal {
 				t.Errorf("condition.Ipv4().DscpSet().State(): got %v, want %v", gnmi.GetConfig(t, dut, condition.Ipv6().DscpSet().Config()), tc.dscpSet)
 			}
 		}
 	}
 
 	// Verify the Classifier is applied on interface by checking the telemetry path state values.
-	classifier := gnmi.OC().Qos().Interface(dp.Name()).Input().Classifier(oc.Input_Classifier_Type_IPV4)
-	if got, want := gnmi.GetConfig(t, dut, classifier.Name().Config()), "dscp_based_classifier"; got != want {
-		t.Errorf("classifier.Name().State(): got %v, want %v", got, want)
-	}
-	if got, want := gnmi.GetConfig(t, dut, classifier.Type().Config()), oc.Input_Classifier_Type_IPV4; got != want {
-		t.Errorf("classifier.Name().State(): got %v, want %v", got, want)
-	}
+
```

Comment:
Two questions here:
1. So we don't need the GetConfig(...) for validation anymore?
2. Please also remove the comment @ 991 if 1) is true.

Reply:
We do validation with  at line 943 with "if got, want := gnmi.GetConfig(t, dut, classifier.Type().Config()), tc.inputClassifierType"